### DISCUSSION TITLE
Help me write an auto-search once you are in the main menu

### DISCUSSION THREAD

**Discussion by PEXCH-SENTXMENT** [danielkrupinski/Osiris#3214] [Ideas]
How do I let the cheat know that you're in the main menu and what command can I use to configure the search

**Comment by danielkrupinski** [comment]
You can check whether you're in the main menu with `bool isInMainMenu = !interfaces->engine->isConnected();`

**Reply by PEXCH-SENTXMENT** [reply]
Thank you. And what commands allow you to run a search?

**Reply by PEXCH-SENTXMENT** [reply]
@danielkrupinski Do you know what commands can be used to do this?
